server/controllers/user.controllers.js

```diff
@@ -121,9 +121,15 @@ const UserController = {
   },
 
   getAllRegisteredUsers(req, res, next) {
-    getAllRegisteredUsers()
-      .then((users) => {
-        res.status(httpStatus.OK).json({ success: true, users });
+    const { page } = req.query;
```

Comment:
This can be combined with the one below:

```suggestion
    const { page, limit } = req.query;
```